libs/vre/pages/search/advanced-search/src/lib/data-access/advanced-search-service/advanced-search.service.ts

```
@@ -243,7 +226,6 @@ export class AdvancedSearchService {
           });
       }),
       catchError(err => {
-        this._handleError(err);
         return []; // return an empty array on error
```

If we catch the error in order to return an empty array we should propagate the error to our AppErrorHandler -> handleError 

this._appErrorHandler.handleError(err);
return of([]);

---

not the scope of this PR